frontend/tet/admin/src/pages/editstatic.tsx

```diff
@@ -14,9 +14,10 @@ import PageNotFound from 'shared/components/pages/PageNotFound';
 import withAuth from 'shared/components/hocs/withAuth';
 import EditorLoadingError from 'tet/admin/components/editor/EditorLoadingError';
 import useLanguageOptions from 'tet-shared/hooks/translation/useLanguageOptions';
+import { Language } from 'shared/i18n/i18n';
 
 const EditStaticPage: NextPage = () => {
-  const { t } = useTranslation();
+  const { t, i18n } = useTranslation();
   const router = useRouter();
```

Comment:
You could use this hook from the shared code:
```
import useLocale from 'shared/hooks/useLocale';
...
const locale = useLocale();
...
const posting = eventToTetPosting(data, keywordResult.getKeywordType, languageOptions,
 locale);
```
It defaults to default language ('fi') when it's missing from the router context. Then you proably dont need helper function inside eventToTetPosting

Reply:
Nwo when I rethink this one more time, you could probably convert `eventToTetPosting` method to hook, `useEventToTetPosting` then you can use `t` and useTranslation inside and dont have to care about the localization. 

  import { useTranslation } from 'next-i18next';
 const useEventToTetPosting = (...) => 
  const { t } = useTranslation();
  return {
     name: t(event.location.name)
   ...
  }
  export default useEventTestPosting
  ```

Reply:
Good suggestion. Changed transfromations to a single hook